src/main/java/com/blogify/blogapi/model/CommentReaction.java

```diff
@@ -0,0 +1,39 @@
+package com.blogify.blogapi.model;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "comment_reaction")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class CommentReaction {
+    @Id
+    private String id;
+
+    /* @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+    // TODO: Add relation with User, waiting for the completion of the User entity */
+
+    @ManyToOne
+    @JoinColumn(name = "comment_id")
+    private Comment comment;
+
+    @Column(name = "is_like")
```

Comment:
not need to spesify @Column(name = "...") when

    the entity have the same name (ex: content = content)
    or if the entity have right conversion (ex: creation_date_time = creationDateTime) all _ change tu upercaset
    so delete @column(name = "...")